app/i18n/es/sub.php

```diff
@@ -127,61 +127,61 @@
 				'xpath' => 'XPath para:',
 			),
 			'json_dotnotation' => array(
-				'_' => 'JSON (dot notation)',	// TODO
+				'_' => 'JSON (sin notación)',
```

Comment:
Could you please double check this translation?

Reply:
I fixed it